gradle/projects.main.properties

```diff
@@ -30,6 +30,7 @@ polaris-quarkus-service=quarkus/service
 polaris-quarkus-server=quarkus/server
 polaris-quarkus-spark-tests=quarkus/spark-tests
 polaris-quarkus-admin=quarkus/admin
+polaris-quarkus-commons=quarkus/commons
```

Comment:
nit: we already have `service/common` ("common" in the singular), any reason to have `quarkus/commons` ("commons" in the plural)?

Reply:
we wanted to share logic between quarkus/admin and quarkus/service hence this module :)

Reply:
I'd suggest to merge the module `admin` and `server`. I didn't see much benefits, while it adds unnecessary burden for dev and release. I will share this idea in the dev mailing list later.

Reply:
I'm talking about naming: why commonS in the plural?